extensions/blocks/eventbrite/eventbrite.php

```diff
@@ -41,7 +41,7 @@ function jetpack_render_eventbrite_block( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'eventbrite' );
 
 	// Show the embedded version.
-	if ( empty( $attr['useModal'] ) ) {
+	if ( empty( $attr['useModal'] ) && 'modal' !== $attr['style'] ) {
```

Comment:
This triggers a notice when the style is not set:

```
Notice: Undefined index: style in wp-content/plugins/jetpack/extensions/blocks/eventbrite/eventbrite.php on line 44
```

Reply:
Fixed in b1d75b0